reactor-core/src/test/java/reactor/core/publisher/FluxFilterWhenTest.java

```diff
@@ -254,13 +254,13 @@ protected void hookFinally(SignalType type) {
 
 	@Test
 	public void cancel() {
-		final FluxIdentityProcessor<Boolean> pp = Processors.multicast();
+		final Sinks.Many<Boolean> pp = Sinks.many().multicast().onBackpressureBuffer();
```

Comment:
not really important but the name of the variable was already quite unclear before and it becomes completely impossible to link to anything now 😅